lib/powerpoint/presentation.rb

```diff
@@ -70,7 +114,10 @@ def save(path)
     end
 
     def file_types
-      slides.map {|slide| slide.file_type if slide.respond_to? :file_type }.compact.uniq
+      types = slides.map do |slide|
+        slide.file_types if slide.respond_to? :file_types
```

Comment:
Use parens unless it's a DSL method (like one on Object or the self object in context)